packages/osd-optimizer/src/worker/webpack.config.ts

```diff
@@ -176,9 +176,8 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
                       )};\n${content}`;
                     },
                     webpackImporter: false,
-                    implementation: require('node-sass'),
+                    implementation: require('sass'),
                     sassOptions: {
-                      outputStyle: 'nested',
```

Comment:
is this because the default is `nested` so this is unneeded?

Reply:
dart-sass didn't support nested. https://github.com/sass/dart-sass/issues/442

Reply:
I see the default for dart-sass is `expanded` is it worth discussing if it should be `compressed`? Pros and cons? Based on the naming of this package as optimization I'd would think `removes as many extra characters as possible` would ideally be the route.

Reply:
I don't see any pros and cons for changing to compressed. will change it to compressed